.github/workflows/main.yaml

```diff
@@ -23,7 +23,7 @@ jobs:
 
     steps:
       - uses: actions/checkout@v2.3.4
```

Comment:
```suggestion
      - uses: actions/checkout@3
```
v3 is out in case we want to upgrade this too

Reply:
<img width="601" alt="Screen Shot 2022-05-12 at 4 55 41 PM" src="https://user-images.githubusercontent.com/8855632/168174511-b38b7fc9-6064-4f68-8f8c-7d256d5dca3a.png">

Reply:
oops apparently I got rid of the v on v3 🫣

Reply:
haha yeah np I just accepted your suggestion and didn't notice 😆